packages/components/src/docs/General/Accessibility.mdx

```diff
@@ -0,0 +1,74 @@
+
+This is an ongoing process. We will continue to improve accessibility and meet at least the **WCAG 2.1 Level AA guidelines** - also with a view to the Barrier-Free Accessibility Strengthening Act that will come into force from 2025.
+
+## Developing a WCAG-compliant Basis
+
+We are committed to creating products that conform to the WCAG 2.1 guidelines, a standard also embraced by AK CD. This commitment includes a range of specific practices:
+
+### Semantic Structure & HTML Usage
+
+- **Native HTML over ARIA labels**: Always use native HTML instead of ARIA labels. This practice ensures that our content is as accessible as possible to a wide range of assistive technologies.
+
+- **Semantically Correct Structure**: Implement a semantically correct structure, for example, H1 followed by H2, and P used for text, etc. This helps assistive technologies understand the structure and meaning of our content.
+
+### Screen reader & keyboard controls
+
+- **Keyboard-Accessible Elements**: All elements should be focusable only with the keyboard, in a desired order and with a visual focus. This assures that our content is accessible to users who rely on keyboard navigation.
+
+- **Preserving ARIA Information**: Do not delete the ARIA information from the solid examples. They ensure correct interpretation by screen readers.
+
+### Multimedia:
+
+- **Alt text**: Provide alternative text for images, icons, and non-text content.
+
+- **Captions and transcripts**: Always include captions and transcripts for videos and audio content.
+
+- **Avoid auto-playing** audio and video without user controls.
+
+### Color:
+
+- **Color contrast**: Verify that color contrast meets the WCAG AA standard.
+- **Color combinations**: Check color combinations for common forms of color blindness.
+
+## Additionally, we incorporate the following practices to ensure accessibility:
+
+- **Mouseless stories** to test and ensure all functionality is accessible without relying on mouse interactions.
+
+- **A11y test cases**.
+
+- **Focus visibility and order** for interactive elements.
+
+## Integration into Processes as Definition-of-Done (DOD)
+
+Accessibility is not an afterthought. It is integrated into our **Definition-of-Done**. This means that no work is considered complete until it meets our accessibility standards.
+
+## The Importance of Web Accessibility
+
+By making your website accessible, you ensure that all users, including people with disabilities, have a good user experience and can easily access information. Web accessibility includes all disabilities that affect access to the Internet, including auditory, cognitive, neurological, physical, speech, visual, and age-related impairments.
+
+In Germany, about 0.4 percent of the population is visually impaired and about 0.1 percent of the inhabitants are auditorily impaired. Making sure our products are accessible opens them up to a wider audience and ensures that we are not excluding any potential users.
```

Comment:
where does this come from? 
Just asking as the 0.5 percent does not really boost the user numbers. I would rather go with the fact that nobody should be excluded and especially as a financial service provider, we cannot withheld this service to some people...

But maybe only a personal point of view.

Reply:
pls also share this with the design team and maybe we need a follow up ticket to align it.

Reply:
Will do!